.bbtravis.yml

```diff
@@ -8,6 +8,7 @@ env:
   matrix:
   - TARGET=qemux86-64
   - TARGET=raspberrypi2
+  - TARGET=intel_nuc-core-i5
```

Comment:
Also currently to make CI working (mainly artifactory push) this value must be the same as MACHINE